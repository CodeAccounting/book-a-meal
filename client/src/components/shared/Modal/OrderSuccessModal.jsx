import React from 'react';
/**
 * @exports
 * @function OrderSuccessModal
 * @returns {component} OrderSuccessModal
 */
const OrderSuccessModal = () => <p className="text-center">Thank you for your order. Your belly will be filled up shortly</p>;

export default OrderSuccessModal;
