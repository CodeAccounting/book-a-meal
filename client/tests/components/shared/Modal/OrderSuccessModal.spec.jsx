import React from 'react';
import OrderSuccessModal from '../../../../src/components/shared/Modal/OrderSuccessModal';

describe('OrderSuccessModal', () => {
  afterAll(() => {
    jest.clearAllMocks();
  });

  it('renders correctly', () => {
    const shallowWrapper = shallow(<OrderSuccessModal />);

    expect(toJson(shallowWrapper)).toMatchSnapshot();
    expect(shallowWrapper.find('p.text-center').text()).toEqual('Thank you for your order. Your belly will be filled up shortly');
  });
});
