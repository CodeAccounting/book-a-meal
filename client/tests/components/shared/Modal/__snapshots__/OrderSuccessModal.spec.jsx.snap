// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderSuccessModal renders correctly 1`] = `
<p
  className="text-center"
>
  Thank you for your order. Your belly will be filled up shortly
</p>
`;
